aristanetworks/ansible-avd
# Update CI to use avd standard docker image

## Types of changes

- [x] Other (please describe): Update Continuous Integration process

## Related Issue(s)

N/A

## Proposed changes

1. Use [`avdteam/base`](https://github.com/arista-netdevops-community/docker-avd-base) image as much as possible in all CI tasks to:

- Reduce configuration overhead
- Do not rely on VM configuration changes
- Validate process in a consistent environment: dev / testing / demo / run
- Make CI easier to read

2. Run __pre-commit__ on every commit instead of custom scripts to make dev process more consistent
3. Update Makefile to remove unnecessary actions.

Same process is going to be applied on [ansible-cvp](https://github.com/aristanetworks/ansible-cvp) repository: https://github.com/aristanetworks/ansible-cvp/pull/208

## How to test

Check [fake PR](https://github.com/titom73/ansible-avd/pull/1) to get results example

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/aristanetworks/ansible-cvp/blob/master/contributing.md#branches) document.
- [x] All new and existing tests passed (`make linting` and `make sanity-lint`).
